src/admin/apiProvider/dataProviders/pitchDataProviders.ts

```diff
@@ -1,67 +1,48 @@
 import _ from "lodash";
 import { DataProvider } from "react-admin";
 
+import { loadProjectPitch, loadProjectPitches, ProjectsPitchesConnection } from "@/connections/ProjectPitch";
 import {
   DeleteV2ProjectPitchesUUIDError,
   fetchDeleteV2ProjectPitchesUUID,
-  fetchGetV2AdminProjectPitches,
   fetchGetV2AdminProjectPitchesExport,
-  fetchGetV2ProjectPitchesUUID,
   fetchPatchV2ProjectPitchesUUID,
-  GetV2AdminProjectPitchesError,
   GetV2AdminProjectPitchesExportError,
-  GetV2ProjectPitchesUUIDError,
   PatchV2ProjectPitchesUUIDError
 } from "@/generated/apiComponents";
 import { ProjectPitchRead } from "@/generated/apiSchemas";
+import { ProjectPitchDto } from "@/generated/v3/entityService/entityServiceSchemas";
 import { downloadFileBlob } from "@/utils/network";
 
-import { getFormattedErrorForRA } from "../utils/error";
-import { apiListResponseToRAListResult, raListParamsToQueryParams } from "../utils/listing";
+import { getFormattedErrorForRA, v3ErrorForRA } from "../utils/error";
+import { raConnectionProps } from "../utils/listing";
 
 export interface PitchDataProvider extends DataProvider {
   export: (resource: string) => Promise<void>;
 }
 
-export const pitchesSortableList: string[] = [
-  "organisation_id",
-  "project_name",
-  "project_objectives",
-  "project_country",
-  "project_county_district",
-  "restoration_intervention_types",
-  "total_hectares",
-  "total_trees",
-  "capacity_building_needs",
-  "created_at",
-  "updated_at",
-  "deleted_at"
-];
+const projectPitchesListResult = ({ data, indexTotal }: ProjectsPitchesConnection) => ({
+  data: data?.map((pitch: ProjectPitchDto) => ({ ...pitch, id: pitch.uuid })) as any,
```

Comment:
I don't think `as any` is needed here; we don't seem to be using it in the other data providers.